Cards/CardContainer.gd

```diff
@@ -393,3 +412,18 @@ func _update_card_positions() -> void:
 		# set position and rotation
 		movement_component.state_properties.desired_position = Vector2(card_x, card_y)
 		movement_component.state_properties.desired_rotation = rotation_amount
+
+func un_queue_card(card : CardWorld):
```

Comment:
wasn't all the queue-ing already done in functions that Cheesy made ?

Reply:
I don't think so, I couldn't find anything like that, and the on click function didn't call anything like that so I don't know

Reply:
@cheesefrycook can you confirm ? I thought I saw something like that in your PR ? or was it a discussion we had but chose to deal with later ?

Reply:
Unqueuing a card previously you had to click the card, so I handle that behavior in _on_card_clicked. So this function is useful if you want to unqueue without clicking I guess.